tooling/api/package.json

```diff
@@ -39,33 +39,31 @@
     "build": "rollup -c --configPlugin typescript",
     "npm-pack": "yarn build && cd ./dist && npm pack",
     "npm-publish": "yarn build && cd ./dist && yarn publish --access public --loglevel silly --tag next",
-    "ts:check": "tsc -noEmit",
-    "lint": "eslint --ext ts \"./src/**/*.ts\"",
-    "lint:fix": "eslint --fix --ext ts \"./src/**/*.ts\"",
+    "ts:check": "tsc",
```

Comment:
any reason to remove `-noEmit`?

Reply:
We already have `"noEmit": true` in `tsconfig.json`

Reply:
hmmm, seems better to leave it here as well, just to not be confused.

Reply:
I have no preference on this, if you prefer that, I'll bring it back